src/components/common.js

```diff
@@ -480,7 +480,7 @@ export const CromwellVersionLink = props => {
       href: `https://github.com/broadinstitute/cromwell/releases/tag/${version}`,
       ...Utils.newTabLinkProps,
       ...props
-    }, ['Cromwell ', version]) :
+    }, ['Cromwell ', version, icon('pop-out', { size: 12, style: { margin: '0 0.25rem 1px' } })]) :
```

Comment:
Let's go back to just setting marginLeft as 0.25rem. When I checked out the code and tried it, it looks OK to me, and it's consistent with all the other usages.

We definitely don't want 0.25rem for marginRight, as that spaces the period out too far.

Thanks!